src/lxc/confile.c

```diff
@@ -4784,6 +4816,38 @@ static int clr_config_net_macvlan_mode(const char *key,
 	return 0;
 }
 
+static int clr_config_net_ipvlan_mode(const char *key,
+				       struct lxc_conf *lxc_conf, void *data)
+{
+	struct lxc_netdev *netdev = data;
+
+	if (!netdev)
+		return -1;
+
+	if (netdev->type != LXC_NET_IPVLAN)
+		return 0;
+
+	netdev->priv.ipvlan_attr.mode = -1;
+
+	return 0;
+}
+
+static int clr_config_net_ipvlan_flags(const char *key,
+				       struct lxc_conf *lxc_conf, void *data)
+{
+	struct lxc_netdev *netdev = data;
+
+	if (!netdev)
+		return -1;
+
+	if (netdev->type != LXC_NET_IPVLAN)
+		return 0;
+
+	netdev->priv.ipvlan_attr.flags = -1;
```

Comment:
Don't you want this to be `0`?

Reply:
@brauner i took my inspiration for this from macvlan https://github.com/lxc/lxc/pull/2950/files/643ac6dc47c5b7820a796937289edc7efae5ca9a#diff-2f3344d3cb9700d11140decc560aaacaR4814 and https://github.com/lxc/lxc/pull/2950/files/643ac6dc47c5b7820a796937289edc7efae5ca9a#diff-2f3344d3cb9700d11140decc560aaacaL4727

I'm guessing this is clearing it to an invalid value, rather than a default value.